generators/generator-base.js

```diff
@@ -2722,4 +2764,40 @@ templates: ${JSON.stringify(existingTemplates, null, 2)}`;
   getR2DBCUrl(databaseType, options = {}) {
     return this.getDBCUrl(databaseType, 'r2dbc', options);
   }
+
+  showHello() {
+    if (this.configOptions.showHello === false) return false;
```

Comment:
is it possible?
``` diff
+ if (!this.configOptions.showHello) return false;
- if (this.configOptions.showHello === false) return false;
```

Reply:
No, the initial version `undefined` should be a positive value.

Reply:
ok